git_machete/git_operations.py

```diff
@@ -337,6 +337,10 @@ def get_remotes(self) -> List[str]:
     def get_url_of_remote(self, remote: str) -> str:
         return self._popen_git("config", "--get", f"remote.{remote}.url").strip()  # 'git remote get-url' method has only been added in git v2.5.1
 
+    def get_organization_and_repository_name_of_remote(self, remote: str) -> Tuple[str, str]:
+        return self._popen_git("config", "--get", f"machete.github.{remote}.organization").strip(), \
+               self._popen_git("config", "--get", f"machete.github.{remote}.repository").strip()
```

Comment:
Actually! we've got `get_config_attr_or_none` above to avoid multiple calls to `git config --get`... pls replace here and wherever else found